test/e2e/routegroup.go

```diff
@@ -390,7 +390,7 @@ rBackend: Path("/backend") -> inlineContent("%s") -> <shunt>;
 			return code == http.StatusTooManyRequests
 		}, false)
 		Expect(err).NotTo(HaveOccurred())
-		Expect(s).To(Equal(expectedResponse))
+		Expect(resp).To(Equal(expectedResponse))
```

Comment:
This fixes the test case for rate limiting, it was never checking the response value of the right request, but the one for the previous tase.

Reply:
Great catch! Thanks for letting me know.

do we need to get the body to  compare?

Reply:
I think my plan was to override `s` but for some reason forgot to.


```diff
		Expect(err).NotTo(HaveOccurred())
-		Expect(resp).To(Equal(expectedResponse))
+		s, err = getBody(resp)
+               Expect(err).NotTo(HaveOccurred())
+		Expect(s).To(Equal(expectedResponse))
```

Reply:
Makes sense, I'll add the change to update `s`.

Reply:
I updated as suggested, thanks 🙂

Reply:
Apparently overriding `s` was wrong 😅  according to @szuecs comment. Will also double check the code locally to confirm.